cli/cli.go

```diff
@@ -20,6 +21,18 @@ type Cli struct {
 	Scanner        *readline.Instance
 }
 
+var instance *Cli
```

Comment:
I am going to remove the support for singleton and move a more functional approach which I feel is more of the idiom of GoLang.